src/app/backend/handler/filter.go

```diff
@@ -177,3 +177,33 @@ func mapUrlToResource(url string) *string {
 	}
 	return &parts[3]
 }
+
+// remoteAddr extracts the remote address of the request, taking into
+// account proxy headers.
+func remoteAddr(r *http.Request) string {
+	if prior := r.Header.Get("X-Original-Forwarded-For"); prior != "" {
```

Comment:
Export those header strings to a custom named string type

Reply:
fixed